autoload/neomake/utils.vim

```diff
@@ -332,7 +332,7 @@ function! neomake#utils#hook(event, context) abort
     if exists('#User#'.a:event)
         let g:neomake_hook_context = a:context
         call neomake#utils#DebugMessage('Calling User autocmd '.a:event
-                                      \ .' with context: '.string(a:context))
+                    \ .' with context: '.string(a:context))
```

Comment:
@davidosomething 
It's only style, not an error?!  AFAIK vint does not handle this, does it?